File: assets/src/testData.js
const bcrypt = require('bcryptjs');

const testUsers = [
    { email: 'rahmaaloui3199@gmail.com', password: bcrypt.hashSync('1234', 10) },
    { email: 'john.doe@example.com', password: bcrypt.hashSync('password1', 10) },
    { email: 'jane.smith@example.com', password: bcrypt.hashSync('password2', 10) },
    { email: 'michael.brown@example.com', password: bcrypt.hashSync('password3', 10) },
    { email: 'emily.jones@example.com', password: bcrypt.hashSync('password4', 10) }
];

module.exports = testUsers;

File: app/testData.py
import bcrypt

testUsers = [
    { 'email': 'rahmaaloui3199@gmail.com', 'password': bcrypt.hashpw('1234'.encode('utf-8'), bcrypt.gensalt()).decode('utf-8') },
    { 'email': 'john.doe@example.com', 'password': bcrypt.hashpw('password1'.encode('utf-8'), bcrypt.gensalt()).decode('utf-8') },
    { 'email': 'jane.smith@example.com', 'password': bcrypt.hashpw('password2'.encode('utf-8'), bcrypt.gensalt()).decode('utf-8') },
    { 'email': 'michael.brown@example.com', 'password': bcrypt.hashpw('password3'.encode('utf-8'), bcrypt.gensalt()).decode('utf-8') },
    { 'email': 'emily.jones@example.com', 'password': bcrypt.hashpw('password4'.encode('utf-8'), bcrypt.gensalt()).decode('utf-8') }
]
